walkthroughs/howto-ingress-enhancements/README.md

```diff
@@ -0,0 +1,437 @@
+# Testing Header/Hostname Matching and Path/Prefix Rewrites
```

Comment:
nit: Replace title with `Walkthrough of matches and rewrites in Virtual Gateway`.

Reply:
done, thank you!